plumbing/protocol/packp/capability/list.go

```diff
@@ -0,0 +1,158 @@
+package capability
```

Comment:
The `List` type will need some more methods in the future:
 - Complementary set: to check the validity of client requests: requested caps not present in the announcement
 - Announced-requested validity: to check client/server compatibility. Example: PushCert without AllowReachableSHA1InWant